checks/kube/deployment.sh

```diff
@@ -20,7 +21,7 @@ usage() {
     echo "  -n NAMESPACE                    Specify the namespace to use"
```

Comment:
wouldn't then namespace be `(default: $NAMESPACE)`?

Reply:
or maybe not, not sure what's the idea behind the env vars defaults below when they actually have a set default